pretrain_prefix_lm.py

```diff
@@ -151,7 +151,7 @@ def get_batch_pipe(data):
         loss_on_targets_only=args.loss_on_targets_only
     )
 
-    return (tokens, position_ids, attention_mask), (labels, loss_mask)
+    return (tokens, position_ids, attention_mask), (labels, loss_mask), prefix_indices
```

Comment:
This is okay as all values beyong the 3rd index are ignored. Which is nice to be used to expose variables needed for testing.